src/MeetingsSDKAdapter.js

```diff
@@ -0,0 +1,153 @@
+import {MeetingsAdapter} from '@webex/component-adapter-interfaces';
+import {Observable, fromEvent, concat} from 'rxjs';
+import {filter, finalize, map, publishReplay, refCount} from 'rxjs/operators';
+
+const LOCAL = 'local';
+const MEDIA_READY = 'media:ready';
```

Comment:
```suggestion
const EVENT_MEDIA_READY = 'media:ready';
```
is a bit more descriptive